src/config/logger.js

```diff
@@ -0,0 +1,36 @@
+const bunyan = require('bunyan');
+let RotatingFileStream = require('bunyan-rotating-file-stream');
+
+let serializer = data => {
+    let query = JSON.stringify(data.query);
+    let options = JSON.stringify(data.options || {});
+
+    return `db.${data.coll}.${data.method}(${query}, ${options});`;
+};
+
+module.exports = bunyan.createLogger({
+    name: 'StylePoint',
+    src: false,
+    streams: [
```

Comment:
Inside `logger.js` there's a configuration of two streams: The first one has the level `info` and the second one (which is the one used for logging to files) has the level `error`. It's as easy as changing the level from `error` to `info` on the second stream and we`ll log all the queries. But I'm not sure about the purpose of logging all the queries in our production environment.